packages/validation-manager/src/ValidationManagerRIP7560.ts

```diff
@@ -25,6 +28,43 @@ export class ValidationManagerRIP7560 implements IValidationManager {
     // TODO
   }
 
+  async _getStakesInfo (operation: OperationBase): Promise<{ senderInfo: StakeInfo, paymasterInfo: StakeInfo, factoryInfo: StakeInfo }> {
+    const senderTMI = await this.stakeManager.getDepositInfo(operation.sender)
+    const senderInfo = {
+      addr: operation.sender,
+      ...senderTMI
+    }
+    let paymasterInfo: StakeInfo = {
+      addr: '',
+      stake: 0,
+      unstakeDelaySec: 0
+    }
+    if (operation.paymaster != null && isAddress(operation.paymaster)) {
```

Comment:
redundant. null is not an address: isAddress checks the value is string and valid as address (only ignoring checksum)